include/swift/Option/Options.td

```diff
@@ -481,6 +481,9 @@ def autolink_force_load : Flag<["-"], "autolink-force-load">,
 def module_abi_name : Separate<["-"], "module-abi-name">,
   Flags<[FrontendOption, ModuleInterfaceOption]>,
   HelpText<"ABI name to use for the contents of this module">;
+def package_name : Separate<["-"], "package-name">,
+  Flags<[FrontendOption, ModuleInterfaceOption]>,
```

Comment:
Please use `ModuleInterfaceOptionIgnorable` instead of `ModuleInterfaceOption` so earlier compiler can compile this emitted interface.